conclave-client/src/main/kotlin/com/r3/conclave/client/EnclaveClient.kt

```diff
@@ -25,7 +25,7 @@ import java.util.concurrent.ScheduledExecutorService
  * Creating a new client instance requires providing an [EnclaveConstraint] object. This acts as an identity of the
  * enclave and is used to check that the client is communicating with the intended one. A private encryption key is also
  * required. There is an constructor overload which creates a random new one for you, or you can use
- * [Curve25519PrivateKey.random] to create a new random one yourself.
+ * [com.r3.conclave.mail.Curve25519PrivateKey.random] to create a new random one yourself.
```

Comment:
This hasn't fixed it: https://r3-conclave.artifacts.teamcity.com/presignedTokenAuth/e39680e6-8043-453c-96e7-f1b36d0a39be/repository/download/ConclaveCore_SdkDocs/9327:id/docs.zip!/api/-conclave%20-core/com.r3.conclave.client/-enclave-client/index.html

Reply:
I could not fix it the right way. Somehow, Dokka doesn't pick it up. As a workaround, I have added the direct link. 

https://r3-conclave.artifacts.teamcity.com/presignedTokenAuth/e8cae3bd-5df3-4511-8427-1eb9a4814f53/repository/download/ConclaveCore_SdkDocs/9368:id/docs.zip!/api/-conclave%20-core/com.r3.conclave.client/-enclave-client/index.html


If there is no other immediate solution, I will add this to the ticket to resolve the @notNull and @Nullable issue.

Reply:
Can you check for all the others as well please.

Reply:
Checked all the changes in TeamCity and fixed wherever the links weren't displayed correctly.